extensions/wikia/Search/classes/IndexService/MediaData.php

```diff
@@ -1,52 +1,57 @@
 <?php
 /**
  * Class definition for \Wikia\Search\IndexService\MediaData
+ *
  * @author relwell
  */
 namespace Wikia\Search\IndexService;
 /**
- * Returns additioanl data about images and videos 
+ * Returns additional data about images and videos
+ *
  * @author relwell
  * @package Search
  * @subpackage IndexService
  */
-class MediaData extends AbstractService
-{
+class MediaData extends AbstractService {
 	/**
 	 * Extracts data from images and videos
+	 *
 	 * @see Wikia\Search\IndexService\AbstractService::execute()
 	 * @return array
 	 */
 	public function execute() {
-		$results = array();
+		$results = [];
 		$service = $this->getService();
-		if (! ( $service->getNamespaceFromPageId( $this->currentPageId ) == NS_FILE
-			    && $service->pageIdHasFile( $this->currentPageId ) ) ) {
+		if ( !( $service->getNamespaceFromPageId( $this->currentPageId ) == NS_FILE &&
+			$service->pageIdHasFile( $this->currentPageId ) )
+		) {
 			return $results;
 		}
-	
-		$detail     = $service->getMediaDetailFromPageId( $this->currentPageId );
-		$metadata   = $service->getMediaDataFromPageId( $this->currentPageId );
+
+		$detail = $service->getMediaDetailFromPageId( $this->currentPageId );
+		$metadata = $service->getMediaDataFromPageId( $this->currentPageId );
 
 		$results['is_video'] = $service->pageIdIsVideoFile( $this->currentPageId ) ? 'true' : 'false';
-		$results['is_image'] = ( ($detail['mediaType'] == 'image') && $results['is_video'] == 'false' ) ? 'true' : 'false';
+		$results['is_image'] =
+			( ( $detail['mediaType'] == 'image' ) && $results['is_video'] == 'false' ) ? 'true' : 'false';
 
-		if (! empty( $metadata ) ) {
+		if ( !empty( $metadata ) ) {
 			$metadata = unserialize( $metadata );
-			$fileParams = array( 'description', 'keywords' );
+			$fileParams = [ 'description', 'keywords' ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "keywords" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "description" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)